### PR TITLE
Move managing organizations and locations (Satellite)

### PR DESCRIPTION
**Pull request by Lennonka**
We're moving these chapters to _Administering_ guide for Sat 6.12:

- Managing organizations
- Managing locations

-------

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3

### CODE REVIEW

**Comment by Lennonka** [comment]
@AkshayGadhaveRH Can you please confirm that the new placement in the _Administering_ guide is suitable?

**Comment by AkshayGadhaveRH** [comment]
Hi @Lennonka the placement of these chapters LGTM. Placing them before `Managing Users and Roles` gives it a sense of flow.

File: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc [hunk 1]
```diff
@@ -10,13 +10,13 @@ For more information, see xref:Creating_a_Custom_File_Type_Repository_{context}[
 * To use HTTPS you require the following certificates on the client:
 +
 . The `katello-server-ca.crt`.
-For more information, see {AdministeringDocURL}Installing_the_Katello_Root_CA_Certificate_admin[Installing the Katello Root CA Certificate] in _{AdministeringDocTitle}_.
+For more information, see {AdministeringDocURL}Installing_the_Katello_Root_CA_Certificate_admin[Installing the Katello Root CA Certificate] in the _Administering {ProjectName}_ guide.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:13]: why remove the attribute for the title?

File: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc [hunk 2]
```diff
@@ -10,13 +10,13 @@ For more information, see xref:Creating_a_Custom_File_Type_Repository_{context}[
...
 . An Organization Debug Certificate.
 ifndef::satellite[]
-For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[Creating an Organization Debug Certificate] in _{ManagingOrganizationsLocationsDocTitle}_.
+For more information, see {ManagingOrganizationsLocationsDocURL}Creating_an_Organization_Debug_Certificate_managing-organizations-locations[Creating an Organization Debug Certificate] in the _Managing Organizations and Locations in {Project}_ guide.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:16]: why remove the attribute?

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc (hunk 1 above)
Comment (Lennonka) [reply guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:13]: By mistake :D Thanks for catching that!

Comment (Lennonka) [reply guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc]: fixed

File: guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc (hunk 2 above)
Comment (Lennonka) [reply guides/common/modules/proc_downloading-files-to-a-host-from-a-custom-file-repository.adoc:16]: fixed

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by Lennonka** [comment]
Merging.

**Pull request merged**